CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.6.1
+
+<!-- generated comparing v5.6.0..master -->
+
+_Apr 11, 2022_
+
+A big thanks to the 8 contributors who made this release possible.  
+This release is mostly about 🐛 bug fixes and 📚 documentation improvements.
```

Comment:
Should we mention something about the docs restructure? I know it has been done technically in the last release but I feel like we should promote it here as well?!

Reply:
Let's use other means of communication for this (Twitter, newsletter). IMO the changelog should be as technical and succinct as possible.
Plus, as you noticed, the changes that you'd like to highlight are not a part of this release.